tests/ChatTests.hs

```diff
@@ -0,0 +1,74 @@
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module ChatTests where
+
+import ChatClient
+import Control.Concurrent.Async (concurrently_)
+import Control.Concurrent.STM
+import Data.Char (isDigit)
+import Data.List (dropWhileEnd, find, isPrefixOf)
+import Simplex.Chat.Controller
+import Simplex.Chat.Types (Profile (..))
+import System.Terminal.Internal (VirtualTerminal (..))
+import Test.Hspec
+
+aliceProfile :: Profile
+aliceProfile = Profile {contactRef = "alice", displayName = "Alice"}
+
+bobProfile :: Profile
+bobProfile = Profile {contactRef = "bob", displayName = "Bob"}
+
+testAddContact :: Spec
+testAddContact = describe "add chat contact" $
+  xit "add contact and send/receive message" $
```

Comment:
`it`

Reply:
it's disabled on purpose, it requires that the server is running and I can't easily start it in CI - it's in a different codebase, so some extra dance in CI is needed to run it here - not planning to do it right now.